ecl/hthor/hthor.cpp

```diff
@@ -10835,12 +10890,13 @@ bool CHThorNewDiskReadBaseActivity::openFilePart(ILocalOrDistributedFile * local
     bool tryRemoteStream = actualDiskMeta->queryTypeInfo()->canInterpret() && actualDiskMeta->queryTypeInfo()->canSerialize() &&
                            projectedDiskMeta->queryTypeInfo()->canInterpret() && projectedDiskMeta->queryTypeInfo()->canSerialize();
 
+
     /*
      * If a file part can be accessed local, then read it locally
      * If a file part supports a remote stream, then use that
      * Otherwise failover to the legacy remote access.
      */
-    const char * format = "thor";   // more - should extract from the current file (could even mix flat and csv...)
+    const char * format = helper.queryFormat();   // more - should extract from the current file (could even mix flat and csv...)
```

Comment:
Is the comment still relevant?

Reply:
yes, because this is getting it from the activity, rather than the dali meta info